lib/pf/Authentication/CreateLocalAccountRole.pm

```diff
@@ -12,11 +12,10 @@ Role that defines the Authentication source behavior for creating local accounts
 
 use Moose::Role;
 use pf::Authentication::constants qw($LOCAL_ACCOUNT_UNLIMITED_LOGINS);
-use pf::ConfigStore::Pf;
 
 has 'create_local_account' => (isa => 'Str', is => 'rw', default => 'no');
 has 'local_account_logins' => (isa => 'Str', is => 'rw', default => $LOCAL_ACCOUNT_UNLIMITED_LOGINS);
-has 'hash_passwords'       => (isa => 'Maybe[Str]', is => 'rw', default => pf::ConfigStore::Pf->new->read("advanced")->{"hash_passwords"});
+has 'hash_passwords' => (isa => 'Maybe[Str]', is => 'rw', default => 'bcrypt');
```

Comment:
@jrouzierinverse: could we use a var in place of 'bcrypt' hard-code value ?

Reply:
Of course we can